package.json

```diff
@@ -30,12 +30,13 @@
   },
   "homepage": "https://github.com/bhovhannes/svg-as-symbol-loader#readme",
   "devDependencies": {
-    "@arkweid/lefthook": "0.7.7",
     "@bhovhannes/shared-config": "0.0.1",
+    "@evilmartians/lefthook": "^0.8.0",
     "@jest/globals": "28.1.0",
-    "jest": "28.1.0",
+    "clear-module": "^4.1.2",
```

Comment:
Why do we need this dependency? I don't see it is used somewhere.

Reply:
We do not, I removed it, thought I did it but apparently I did not.